From 0f8290c943da298abd269ca60fd8375dfb219971 Mon Sep 17 00:00:00 2001
From: Alexandru DAMIAN <alexandru.damian@intel.com>
Date: Thu, 12 Jul 2012 12:54:48 +0300
Subject: [PATCH] Fixing keyboard_force_release.sh shell script path

With the introduction of rootprefix, the keyboard-force-release.sh.in
was modified to be executed with @rootprefix@/bin/sh, which is wrong
because @rootprefix@ defaults to /usr (which is correct), but the
shell is always at /bin/sh (IEEE Std 1003.2-1992).

Therefore the interpreter for shell scripts needs to be /bin/sh at all times.

The upstream moved to configurable root prefix, this patch taclkes a 
transition bug, and will not be applied upstream.

Upstream-Status: Inappropriate [legacy version]

Signed-off-by: Alexandru DAMIAN <alexandru.damian@intel.com>
---
 src/keymap/keyboard-force-release.sh.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/keymap/keyboard-force-release.sh.in b/src/keymap/keyboard-force-release.sh.in
index dd040ce..597a3a6 100755
--- a/src/keymap/keyboard-force-release.sh.in
+++ b/src/keymap/keyboard-force-release.sh.in
@@ -1,4 +1,4 @@
-#!@rootprefix@/bin/sh -e
+#!/bin/sh -e
 # read list of scancodes, convert hex to decimal and
 # append to the atkbd force_release sysfs attribute
 # $1 sysfs devpath for serioX
-- 
1.7.9.5

